packages/theme-chalk/src/alert.scss

```diff
@@ -14,7 +14,7 @@
   --el-alert-icon-size: 16px;
   --el-alert-icon-large-size: 28px;
 
-  @each $type in (success, info, warning, danger) {
+  @each $type in (success, info, warning, error) {
```

Comment:
I think we can apply some redundant here to allow `danger` and `error` to exist at the same time since for uses in mainland China we tend to use `error` but for users outside mainland China, `danger` would be more syntaxially appropriate.

Reply:
Is alert special, or all components?

Reply:
alert `type` is `success/warning/info/error`.

If we want to add `danger`, that means we should add `danger` here too.

It may not be necessary.